spec/requests/api/v1/events/events_spec.rb

```diff
@@ -82,7 +76,27 @@
       expect(response).to match_response_schema(:event)
     end
 
-    it 'returns and error message when invalid' do
+    it 'returns an error when the event cant be found' do
+      invalid_id = 666
```

Comment:
Eh, I'm not a fan of making a call the the DB when you don't have to. Does `foo` not work? Or even `nil`?